parent/pom.xml

```diff
@@ -18,7 +18,7 @@
   </description>
 
   <properties>
-    <version.quarkus>3.8.5</version.quarkus>
+    <version.quarkus>3.14.1</version.quarkus>
```

Comment:
Latest Quarkus [version](https://mvnrepository.com/artifact/io.quarkus/quarkus-core/3.14.1)

Reply:
They released `3.14.2` yesterday.
https://github.com/quarkusio/quarkus/releases/tag/3.14.2

Reply:
It's available on [Maven Central](https://central.sonatype.com/artifact/io.quarkus/quarkus-bom/3.14.2) now as well. I think we should update to this version directly.

Reply:
Thank you both, [bumped](https://github.com/camunda/camunda-bpm-platform/pull/4563/commits/30d74b71b18b9ea8ad2bc8bacf90ee48b7186267) the version to `3.14.2`